core/src/main/java/tech/pegasys/web3signer/core/Runner.java

```diff
@@ -103,11 +108,12 @@ public void run() {
     try {
       createVersionMetric(metricsSystem);
       metricsEndpoint.start(vertx);
-
+      boolean failToLoad = false;
```

Comment:
rename suggestion: failToLoad -> failedToLoadAnySigners (or is it failedToLoadSomeSigners?)

Reply:
failed to load some signers maybe? as long as there is an error when loading from one source, it will be set to true.

Reply:
Updated.